src/web/WebLogService.cpp

```diff
@@ -158,7 +158,7 @@ void WebLogService::loop() {
 // convert time to real offset
 char * WebLogService::messagetime(char * out, const uint64_t t) {
     if (!time_offset_) {
-        strcpy(out, uuid::log::format_timestamp_ms(t, 3).c_str());
+        strlcpy(out, uuid::log::format_timestamp_ms(t, 3).c_str(), sizeof(out));
```

Comment:
Same sizeof issue here, to check you'll need to pass the size to the messagetime-function

Reply:
or use strlen(out) ?

Reply:
No, declare as `char * WebLogService::messagetime(char * out, const uint64_t t, size_t  bufsize) {`, than you can use 
`strlcpy`. 
You have to determine the size where the busser is declared. `char buffer[20]` have sizeof(buffer) as 20, but`char * pbuffer=buffer` gives sizeof(pbuffer) with result 4. strlen is undefined if the buffer is not initialized.

Reply:
fixed